### ISSUE TITLE
docker is killing "hyper-v virutal machine bus" and NICs

### ISSUE THREAD

**Issue by ormico** [docker/for-win#1409]
### Expected behavior
I'm trying to get Docker For Windows Running on a Windows 10 Pro VM running on Hyper-V on Windows 2016.

The purpose is to run a CI Build agent so that I can build Visual Studio 2017 projects that have Docker support.

### Actual behavior
When Docker for Windows is installed, the VM's NICs will disappear and if you look in Device Manager "hyper-v virutal machine bus" will show to be in an error state.

Uninstalling Docker for Windows resolves the issue. Re-installing will cause the error to come back.

For a while I could work around the issue by removing the Virtual NIC and re-adding it, but that solution stopped working.

### Information
I cannot upload a Diagnostic since I cannot get Network access and Docker running at the same time.
  - host distribution and version (Windows version, build number, etc)
This is a Hyper-V Gen 1 Config ver. 8 VM running on Hyper-V on Windows 2016. The Windows version of the VM is Version 1703 (OS Build 15063.726)

I'll try this on a Gen 2 VM later.

**Comment by ormico** [comment]
Docker version is 17.09.1-ce, build 19e2cf6

**Comment by ormico** [comment]
I tried this on a Gen2 VM and I am not experiencing the same problem, so far.

**Comment by ormico** [comment]
some additional info. The Gen2 VM is still fine and I'm using docker on it.
the Gen1 VM however got worse. I uninstalled Docker and managed to get the NIC to come back for a while but then the NIC disappeared again and nothing I have tried has helped. I was able to get Device Manager to show all of the hidden NICs (remnants of all the NICs I had previously added and removed through Hyper-V manager) and I deleted them but that didn't help either. 

Currently the VM is only accessable through the Hyper-V manager as I cannot RDP into it w/o a NIC.

Since I didn't take a snapshot of the VM before installing Docker my only option is to restore from backup but it's probably easier to just create a new VM and set it up as a Gen 2.

Don't know if anyone else has had any issues like this or just me. Hope this helps
